CHANGELOG.md

```diff
@@ -26,6 +26,12 @@ Classify the change according to the following categories:
     ##### Removed
     ### Patches
 
+## Develop
+### Minor Updates
+#### Added
+- `/erp/inputs` endpoint (calls `erp_help()`, same as `/erp/help`)
```

Comment:
@hdunham is it typical to have two endpoints return the same thing? And is that a correct understanding, that erp/inputs and erp/help will both return the input fields (not the actual user input values) for the ERP function?

Reply:
Yeah that's what we already do for the regular /help and job/inputs endpoints. At some point we could change help to return the info for both inputs and outputs, but for this I wanted a quick solution so I just followed the existing pattern.

Reply:
ok cool, I haven't worked with the endpoints much so was curious